api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -403,6 +403,68 @@ public ResponseEntity<CohortReviewWithCountResponse> getParticipantCohortStatuse
                         cohortReview.getCohortReviewId(), pageRequest)));
   }
 
+  /**
+   * Get all participants for the specified cohortId and cohortReviewId. This endpoint does
+   * pagination based on page, pageSize, sortOrder and sortColumn.
+   */
+  @Override
+  public ResponseEntity<CohortReviewWithCountResponse> getParticipantCohortStatuses(
+      String workspaceNamespace,
+      String workspaceId,
+      Long cohortId,
+      Long cohortReviewId,
+      PageFilterRequest request) {
+    DbWorkspace dbWorkspace =
+        workspaceAuthService.getWorkspaceEnforceAccessLevelAndSetCdrVersion(
+            workspaceNamespace, workspaceId, WorkspaceAccessLevel.READER);
+    long cdrVersionId = dbWorkspace.getCdrVersion().getCdrVersionId();
+
+    CohortReview cohortReview;
+    List<ParticipantCohortStatus> participantCohortStatuses = new ArrayList<>();
+    DbCohort cohort = cohortReviewService.findCohort(dbWorkspace.getWorkspaceId(), cohortId);
+    PageRequest pageRequest = createPageRequest(request);
+    convertGenderRaceEthnicitySortOrder(pageRequest);
+
+    try {
+      cohortReview =
+          cohortReviewService.findCohortReviewForWorkspace(cohort.getCohortId(), cohortReviewId);
+      participantCohortStatuses =
+          cohortReviewService.findAll(cohortReview.getCohortReviewId(), pageRequest);
+    } catch (NotFoundException nfe) {
+      cohortReview = cohortReviewService.initializeCohortReview(cdrVersionId, cohort);
+    }
```

Comment:
The try/catch can be removed since the new implementation is just a straight read. Will no longer need to initialize since cohort review will always exist.
```suggestion
    cohortReview =
        cohortReviewService.findCohortReviewForWorkspace(cohort.getCohortId(), cohortReviewId);
    participantCohortStatuses =
        cohortReviewService.findAll(cohortReview.getCohortReviewId(), pageRequest);
```